File: go-sdk/pkg/models/operations/getfollowedtags.go
package operations

import (
	"github.com/speakeasy-sdks/forem-sdks/go-client-sdk/pkg/models/shared"
)

type GetFollowedTagsResponse struct {
	ContentType  string
	FollowedTags []shared.FollowedTag
	StatusCode   int64
}

File: python-sdk/src/foremapi/models/operations/getfollowedtags.py
import dataclasses
from typing import Optional
from ..shared import followedtag as shared_followedtag


@dataclasses.dataclass
class GetFollowedTagsResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    followed_tags: Optional[list[shared_followedtag.FollowedTag]] = dataclasses.field(default=None)
    
